index.html

```diff
@@ -20,8 +20,12 @@
               <img src="images/MyLogo.svg" alt="Logo" />
             </a>
             <nav class="dropdown">
-              <img src="images/NavMenu.svg" alt="Nav Menu" class="nav_menu" />
-              <ul class="mobile_nav">
+                <img src="images/NavMenu.svg" alt="Nav Menu" class="nav_menu" />
+              <ul class="mobileNav">
```

Comment:
- [ ] Please make sure the mobile menu follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins). Currently, your `design color` for text is not` white color` like it in the Figma design. Take a look at the Figma style before making necessary changes to styling 👇 
- Figma design color 👇 
![Screenshot (208)](https://user-images.githubusercontent.com/67087664/156146988-c3a2fa25-d5cf-4ab4-afb8-3363218e2296.png)
- Yours 👇 
![Screenshot (209)](https://user-images.githubusercontent.com/67087664/156147163-065fa92c-444c-45b5-a3d7-4b7c5811af89.png)

Reply:
Done. I've changed the colors. I really forgot about them, thank you very much for pointing this out.